AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Models/Features.swift

```diff
@@ -8,25 +8,26 @@
 import Foundation
 import AWSCLIUtils
 
-struct FeaturesReader: Decodable {
-    private let requestFilePath: String
-    private let mappingFilePath: String
```

Comment:
Instead of taking individual paths to files, this type takes the caller's `repoPath` as a dependency & assumes the individual files are in the repo's root.